lib/countly-storage.js

```diff
@@ -0,0 +1,168 @@
+const fs = require('fs');
+const path = require('path');
+var cc = require("./countly-common");
+var storagePath;
+var __data = {};
+
+var setStoragePath = function (path) {
+    defaultPath = "../data/"; // Default path
```

Comment:
what was the default path before?
was it ../bulk_data/

Reply:
well this is ok but countly and bulk-data defaults are different
If it is not a problem then all cool